### PR TITLE
Exporting data migration records

### PR DESCRIPTION
**Pull request by hectorcorrea**
Sort of automates the process to export and import the test data from one instance to another. This is heavily based on the steps indicated https://github.com/pulibrary/pdc_describe/blob/main/docs/test_data_in_production.md

The general workflow is:

```
# Export the data to JSON files 
DATA_MIGRATION=true bundle exec rspec ./spec/system/data_migration

# Delete existing data (optional)
bundle exec rake works:delete_user_works[your-net-id]

# Import the data from the JSON files
bundle exec rake works:import_works[./tmp/data_migration]
```

Work towards https://github.com/pulibrary/pdc_discovery/issues/340

### CODE REVIEW

File: lib/tasks/works.rake [hunk 1]
```diff
@@ -58,4 +58,27 @@ namespace :works do
...
+    path = File.join(args[:path], "*.json")
+    approver = User.first
+    puts "Importing files from: #{path}"
+    Dir.glob(path).each do |file_name|
+      hash = JSON.parse(File.read(file_name))
+      resource = PDCMetadata::Resource.new_from_json(hash)
+      work = Work.new(resource: resource)
+      work.collection = Collection.research_data
+      work.created_by_user_id = approver.id
+      work.state = "approved"
```
Comment (hectorcorrea) [inline lib/tasks/works.rake:79]: Set to approved for now because that is what I need to get the records to show in the RSS feed but we could tweak this.

**Comment by hectorcorrea** [comment]
@bess I sort of automated the process to export/import the data from the tests based on your instructions in the docs folder. This is something that I am using as I work on https://github.com/pulibrary/pdc_discovery/issues/340 and I thought could be useful in general.

File: spec/system/data_migration/baldwin_form_submission_spec.rb [hunk 2]
```diff
@@ -64,6 +64,8 @@
       # This will allow us to evolve our local datacite standards and test our records against them.
       datacite = PDCSerialization::Datacite.new_from_work(baldwin_work)
       expect(datacite.valid?).to eq true
+
+      export_spec_data("baldwin.json", baldwin_work.to_json)
```
Comment (hectorcorrea) [inline spec/system/data_migration/baldwin_form_submission_spec.rb:68]: Notice that by default the call to `export_spec_data` does nothing, it only exports the data if `ENV["DATA_MIGRATION"] == "true"`

**Review by bess — APPROVED** [review]
🤩

**Pull request merged**